Tools/clinic/clinic.py

```diff
@@ -1455,6 +1449,14 @@ def parser_body(
         if compiler_warning:
             parser_definition = compiler_warning + "\n\n" + parser_definition
 
+        # Copy converters includes at the end, since includes can be
+        # added late
+        for converter in converters:
+            include = converter.include
+            if include is not None:
```

Comment:
I think that can be made an assert.

Reply:
Not all converters need a special include. Do you mean that `include` variable is never None?